packages/currency-utils/src/currencyUtils.ts

```diff
@@ -28,25 +22,56 @@ export const getBalance = async ({
   }
 };
 
-export const getOraclePrice = async ({
-  tokenType,
-  cluster,
-  connection,
-}: {
-  tokenType: "HNT";
-  cluster: Cluster;
-  connection: Connection;
+export const PYTH_HERMES_URL = "https://hermes.pyth.network/"
+const HNT_PRICE_FEED_ID = "0x649fdd7ec08e8e2a20f425729854e90293dcbe2376abc47197a14da6ff339756"
```

Comment:
Nit: move to constants.ts of spl-utils? and replace here and in `mintDataCredits.ts`